package.json

```diff
@@ -14,7 +14,6 @@
     "@mui/lab": "^5.0.0-alpha.175",
     "@mui/material": "^5.17.1",
     "@mui/styles": "^5.17.1",
-    "@pixi/unsafe-eval": "^7.4.3",
```

Comment:
Removing this dependency broke the built version of the webapp (the graph is not visible in the Simulation view), with this error in the console:
> Uncaught (in promise) Error: Current environment does not allow unsafe-eval, please use pixi.js/unsafe-eval module to enable support.

Reply:
pixi/unsafe-eval is now directly integrated with PixiJS v8